### ISSUE TITLE
WIW is not a constructor

### ISSUE THREAD

**Issue by dj21d** [jgr3go/node-wheniwork#1]
I'm getting the following error when attempting to use your module:
```
./getWIWSchedule.js:3
let wiw = new WIW("apikey", "username", "password");
          ^

TypeError: WIW is not a constructor
    at Object.<anonymous> (./getWIWSchedule.js:3:11)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.runMain (module.js:604:10)
    at run (bootstrap_node.js:389:7)
    at startup (bootstrap_node.js:149:9)
    at bootstrap_node.js:504:3
```

This is a copy paste of your basic usage into node. Any help?

**Comment by jgr3go** [comment]
The documentation wasn't updated when I released v2.0, sorry about that.  In order to support Typescript it's no longer the default export, so you instead have to 

```
let WIW = require('wheniwork').WhenIWorkApi;
let wiw = new WIW(...);
```

The docs have been updated.  You should upgrade to the newest version as well (2.0.3), otherwise you'll have to annoyingly require `require('wheniwork').default;` instead of `.WhenIWorkApi`;

**Comment by jgr3go** [comment]
Closing this issue since it's been updated. Feel free to ask if you have any other questions!
